tests/test_hep_bd0xx.py

```diff
@@ -795,6 +795,39 @@ def test_report_numbers_from_037__z_9():
     assert expected == result['037']
 
 
+def test_report_numbers_from_037__a_9_arXiv_reportnumber():
+    schema = load_schema('hep')
+    subschema = schema['properties']['report_numbers']
+
+    snippet = (
+        '<datafield tag="037" ind1=" " ind2=" ">'
+        '  <subfield code="9">arXiv:reportnumber</subfield>'
+        '  <subfield code="a">LIGO-P1500247</subfield>'
+        '</datafield>'
+    )  # record/1618037
+
+    expected = [
+        {
+            'source': 'arXiv',
+            'value': 'LIGO-P1500247',
+        },
+    ]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['report_numbers'], subschema) is None
+    assert expected == result['report_numbers']
+
+    expected = [
+        {
+            '9': 'arXiv:reportnumber',
+            'a': 'LIGO-P1500247',
+        },
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['037']
```

Comment:
Can you add an assertion about `035` ? Otherwise I don't think I will remember what was the problem here...